tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/os/UserTaskZeebeRecordProcessorOpenSearch.java

```diff
@@ -124,4 +128,22 @@ private BulkOperation getVariableQuery(final TaskVariableEntity variable)
           e);
     }
   }
+
+  private UserTaskListViewEntity createTaskListViewInput(final TaskEntity taskEntity) {
+    return new UserTaskListViewEntity(taskEntity);
+  }
+
+  private BulkOperation persistUserTaskToListView(
+      final UserTaskListViewEntity tasklistListViewEntity) {
+    return new BulkOperation.Builder()
+        .update(
+            up ->
+                up.index(tasklistListViewTemplate.getFullQualifiedName())
+                    .id(tasklistListViewEntity.getId())
+                    .document(CommonUtils.getJsonObjectFromEntity(tasklistListViewEntity))
```

Comment:
you need to use `userTaskRecordToTaskEntityMapper` the same way you did for ElasticSearch importer